lib/dev/frame-script.js

```diff
@@ -72,6 +72,9 @@ addMessageListener("sdk/event/message", onInMessage);
 addMessageListener("sdk/port/message", onInPort);
 
 const observer = {
+  handleEvent: ({target, type}) => {
+    this.observe(content.document, type);
```

Comment:
> I'm guessing that this.observe is undefined here? @gozala does that make sense? and we should have a test that this works.

You are right! I had method before just changed it to arrow to match `observe` & have not run tests again. I'll update that.

Yes we do have tests for unload that were passing until I've changed method to arrow function.